src/arith.rs

```diff
@@ -374,6 +497,21 @@ impl U256 {
     pub fn bits(&self) -> BitIterator {
         BitIterator { int: &self, n: 256 }
     }
+
+    pub const fn from_le_slice(bytes: &[u8]) -> Self {
+        assert!(bytes.len() == 32);
+        let mut lo = [0u8; 16];
+        let mut hi = [0u8; 16];
+        let mut idx = 0;
+
+        while idx < 16 {
+            lo[idx] = bytes[idx];
+            hi[idx] = bytes[idx + 16];
```

Comment:
Probably not, but wondering if this could be faster if copying by slice rather by each u8? Probably a not worth micro-optimization, just sharing thought

Reply:
I'm not very familiar with the requirements for `const` functions and I wasn't sure how to do it by slice in `const`; I also figured performance wouldn't particularly matter since this is a `const` function that's only intended for initializing `const`s (although in practice, when I've tried to switch out `const` initialization from `mem::transmute` to this I have (oddly) seen runtime performance hits, so I guess that logic doesn't quite hold).